### ISSUE TITLE
Add a CHANGELOG.md 

### ISSUE THREAD

**Issue by Ademsk1** [animo/animo-mediator#19]
Would be awesome if we could get a CHANGELOG file to keep track of the changes being made

**Comment by TimoGlastra** [comment]
Yes thanks for opening the issue. We'll make sure to create a changelog before releaseing v0.2.0
